openstudiocore/src/measure/OSArgument.cpp

```diff
@@ -38,7 +38,7 @@
 #include "../utilities/core/Assert.hpp"
 #include "../utilities/core/Containers.hpp"
 #include "../utilities/core/Compare.hpp"
-#include "../utilities/core/Json.hpp"
+#include "../utilities/core/QJson.hpp"
```

Comment:
@jmarrec is the idea that this file will be removed or will it hang around?  I don't think we should have a file named QJson hanging around for a long time.